gen-cpp/CppRacconChat.h

```diff
@@ -0,0 +1,18 @@
+#include "RacconChat.h"
```

Comment:
move these files out of gen-cpp. Give it a better name, it shouldn't have Cpp in it. maybe RacoonChatServer or something